db/schema.rb

```diff
@@ -326,6 +326,7 @@
     t.string "gender_identity"
     t.string "sexual_orientation"
     t.boolean "user_defined_symptom_onset"
+    t.date "extended_isolation"
```

Comment:
This migration will go in before the extended isolation migration, extended isolation won't be going until after the patch, so this would need to be updated.

Reply:
Thanks for catching this, I think when I was switching between branches, the migration file from the extended_isolation branch did not get deleted so when I ran db:migrate, it included this field.